test/correctness/rfactor.cpp

```diff
@@ -17,27 +17,59 @@ typedef map<string, vector<string>> CallGraphs;
 
 class CheckCalls : public IRVisitor {
```

Comment:
I actually think this PR would be a significant net *reduction* in code if it weren't for this :)